Project 12/src/main/java/calculator/CalculatorLogic.java

```diff
@@ -0,0 +1,34 @@
+package calculator;
+
+import java.util.Scanner;
+
+public class CalculatorLogic {
```

Comment:
why do you need this class in homework about unit tests?
also please remove other unnecessary classes